pointsource_imaging/MIRI_Aperture_Photometry_solution.ipynb

```diff
@@ -161,6 +161,18 @@
     "### 2.2<font color='white'>-</font>Plot configuration<a class=\"anchor\" id=\"plot\"></a> ###"
```

Comment:
<p>The problem here is that you're slicing <code>mask</code> with "<code>==</code>" when it isn't needed.</p><p> </p><p>To assign all NaN pixels to True, try <code>mask[np.isnan(img_F560W.data)] = True</code>.</p><p> </p><p>To assign all infinite pixels to True, try <code>mask[~np.isfinite(img_F560W.data)] = True</code> (the tilde flips True to False and vice versa).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
Done